RELEASE_HISTORY.md

```diff
@@ -0,0 +1,73 @@
+# Zowe CLI Releases
+
+Zowe follows a regular release schedule with major versions released every two years and minor versions approximately every six weeks. Full details regarding the Zowe release schedule are available on [zowe.org](https://www.zowe.org/download#timeline) and the Zowe Community [Github](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/release.md).
+
+## Zowe Release Schedule
+
+### Zowe v3.x LTS Releases
+|  Version | Release Date |  Status    | Release Notes |
+|:--------:|:------------:|:----------:|:-------------:|
+| v3.0.0 | 2024-07-19 | **Active** | [Release Notes](https://docs.zowe.org/stable/whats-new/release-notes/v3_0_0) |    |
```

Comment:
1. Should we make the date less specific (e.g. Sep 2024) since the exact release date often gets delayed?
2. I assume the "Active" was an example of what this table would look like in the future 😋 Until we have our first V3 release, should the status say "Pending" for now?
3. There seems to be an extra empty column at the end
```suggestion
| v3.0.0 | Sep 2024 | **Pending** | [Release Notes](https://docs.zowe.org/stable/whats-new/release-notes/v3_0_0) |
```

Reply:
oops i thought i was already being less specific with the date thank you. an oversight on my end